src/System.Security.Cryptography.Pkcs/src/Resources/Strings.resx

```diff
@@ -241,6 +241,30 @@
   <data name="Cryptography_NotValidPublicOrPrivateKey" xml:space="preserve">
     <value>Key is not a valid public or private key.</value>
   </data>
+  <data name="Cryptography_Pkcs12_CannotProcessEncryptedSafeContents" xml:space="preserve">
+    <value>This operation is not valid on an encrypted or enveloped Pkcs12SafeContents.</value>
+  </data>
+  <data name="Cryptography_Pkcs12_CertBagNotX509" xml:space="preserve">
+    <value>The Pkcs12CertBag contents are not an X.509 certificate.</value>
+  </data>
+  <data name="Cryptography_Pkcs12_PfxIsSealed" xml:space="preserve">
+    <value>The Pkcs12Builder has been sealed and can no longer be modified.</value>
+  </data>
+  <data name="Cryptography_Pkcs12_PfxMustBeSealed" xml:space="preserve">
+    <value>The Pkcs12Builder must be sealed before it can be encoded.</value>
```

Comment:
Is "sealed" the language the RFC uses?  If yes, ok.  If no, can we pick a different term?  For someone who knows C#, this makes it sounds like the class itself is or must be `sealed`.

Reply:
> Is "sealed" the language the RFC uses?

No, it's a consequence of the builder API.  It means "you need to call one of SealWithMac, SealWithoutIntegrity, or the future SealWithCertificate" (using the verb from the method name)

Reply:
Should we consider an alternative to "Seal" then?  "Freeze"?

Reply:
Freeze feels less concise (and "Seal" matches the vault/safe metaphor of PFX better).  Some thesaurus time turned up "Stamp" (which definitely works for MAC and Certificate, but feels wrong for "WIthoutIntegrity").  "Close" would be good, if that didn't already have implied meaning.  "Shut"?

Alternatively, if the problem is just the resource strings then the wording could be adjusted.  "One of the Seal methods must be called before calling an Encode method".

Reply:
Ok.  Let's at least reword the resource strings. Thanks.